src/CMakeLists.txt

```diff
@@ -1045,3 +1045,6 @@ add_subdirectory(script)
 if(WITH_EMBEDDED)
   add_subdirectory(libcephd)
 endif()
+
+configure_file(${CMAKE_SOURCE_DIR}/src/upstart/ceph-osd.conf.in
```

Comment:
> I suppose `${CMAKE_RUNTIME_OUTPUT_DIRECTORY}` only includes executables? Or configs are acceptable too?

yeah, you are right. `${CMAKE_RUNTIME_OUTPUT_DIRECTORY}` is for runtime files only. and libraries are also located in it.